atd-vze/src/views/Crashes/Crash.js

```diff
@@ -117,29 +117,37 @@ function Crash(props) {
     <div className="animated fadeIn">
       <Row>
         <Col>
-          <h2 className="h2 mb-3">{`${primaryAddress} & ${secondaryAddress}`}</h2>
+          {primaryAddress && secondaryAddress ? (
+            <h2 className="h2 mb-3">{`${primaryAddress} & ${secondaryAddress}`}</h2>
+          ) : (
+            <h2 className="h2 mb-3">{`Location Not Yet Assigned`}</h2>
```

Comment:
I don't think this is the accurate way to describe it and might lead to confusion. A location association is different from having missing address fields. 

What if we did this instead:

```js
<h2 className="h2 mb-3">
  {`${primaryAddress ? primaryAddress : "PRIMARY ADDRESS MISSING"} & ${secondaryAddress ? secondaryAddress : "SECONDARY ADDRESS MISSING}`}
</h2>
```

Reply:
I'm glad you pointed this out, because it does make it sound like it has something to do with the Location view. I just fixed this in #412 so it will be clearer.